WooCommerce/UITestsFoundation/Screens/Products/SingleProductScreen.swift

```diff
@@ -41,17 +41,32 @@ public final class SingleProductScreen: ScreenObject {
         return self
     }
 
-    public func verifyNewProductScreenLoaded(productName: String) {
+    public func verifyPublishedProductScreenLoaded(productType: String, productName: String) {
+        let priceLabel = NSPredicate(format: "label == 'Price'")
+        let addVariationLabel = NSPredicate(format: "label == 'Add variations'")
+
+        // common fields on a published product screen
         XCTAssertTrue(app.buttons["save-product-button"].exists)
         XCTAssertTrue(app.staticTexts["TIP"].exists)
         XCTAssertTrue(app.textViews[productName].exists)
+
+        // different product types displays different fields on the published product screen
```

Comment:
```suggestion
        // different product types display different fields on the published product screen
```

Reply:
updated in https://github.com/woocommerce/woocommerce-ios/pull/9441/commits/ecbb4b636a6e57c65da9de3e1d38b534adc8d4fa (also fixed another similar one)